demo/scripts/teardown.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Copyright IBM Corp All Rights Reserved
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+DEMO_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+DEMO_DOCKER_COMPOSE=${DEMO_SCRIPT_DIR}/../docker-compose.yml
+
+# SCRIPT_DIR is the docker compose script dir that needs to be defined to source environment variables from the FPC Network
+SCRIPT_DIR=${DEMO_SCRIPT_DIR}/../../utils/docker-compose/scripts
+. ${SCRIPT_DIR}/lib/common.sh
+
+COMPOSE_IGNORE_ORPHANS=true docker-compose -f ${DEMO_DOCKER_COMPOSE} kill && COMPOSE_IGNORE_ORPHANS=true docker-compose -f ${DEMO_DOCKER_COMPOSE} down
+
+"${SCRIPT_DIR}/teardown.sh" --clean-slate 
```

Comment:
The only reason I do clean-slate is because I run generate right below, and we need to get rid of the CA volumes if we recreate the certs. I could separate out the generate and then add the flag here too if you prefer.

Reply:
i don't think this is true (wipe ca volumes when generating new (client) certs: i think we had the same discussion on the prior PR and whatever we discussed then i did test and CA wipe was not necessary ...

Reply:
Oh I must have misunderstood that conversation. I understood it as the CA volumes only need to be wiped if the certificates have changed. Then in that case I can remove the flag. That was my only reason for adding the `clean-slate`

Reply:
Added a `clean-slate` flag to the demo teardown script. When the flag is added it will run the FPC network teardown with the clean-slate and get rid of the mockcc chaincode image as well.